content/en/integrations/guide/source-code-integration.md

```diff
@@ -28,72 +24,60 @@ With the source code and GitHub Apps integrations, you can also see inline code
 
 ## Configuration
 
-To map telemetry data with your source code, you need to send metadata to Datadog from your CI pipeline.
-
-To tag your process with the commit SHA, you need a container label, Kubernetes label, or annotation.
+<div class="alert alert-info">
+The source code integration supports Go and all JVM languages.
+<br>
+You need to be using Datadog Agent 7.33.0 or higher.
+</div>
 
-1. Set a container tag or extend the `DD_TAGS` environment variable.
-2. Upload your git metadata including the commit SHAs and your git repository URL by running [`datadog-ci git-metadata upload`][1].
-3. Optionally, [install GitHub Apps][2].
+To map telemetry data with your source code, you need to:
 
-Datadog correlates the places where you can link directly to your git repository.
+1. Add `git.commit.sha` and `git.repository_url` tags to your containers, or directly on your telemetry.
+2. Upload metadata about your git repository by running [`datadog-ci git-metadata upload`][1] in your CI pipeline.
+3. Optionally, [install a GitHub App][2] to display inline source code snippets.
 
-### Tag your builds
+### Tag your telemetry
 
-To get direct links from your stacktrace to your git repository, tag your telemetry with a `git.commit.sha` tag.
+To link data to a specific commit, tag your telemetry with a `git.commit.sha` tag.
 
 {{< tabs >}}
-{{% tab "Docker Runtime" %}}
-
-If your containers are running on Docker, Datadog can extract the commit SHA directly from your images' Docker labels.
+{{% tab "Containers" %}}
```

Comment:
Changing this to "Containers" doesn't seem quite right, since Kubernetes technically also deals with containers.

Reply:
I thought about changing this to "Docker & containerd", but Kubernetes can also be using both these container runtimes; my reasoning was that "Containers" next to "Kubernetes" would naturally be understood. I don't know if there is a term we generally use to designate non-k8s containers in other places of the documentation that could apply here ?

Reply:
I just checked in with our docs expert, @ceciliawatt and it sounds like we should keep this tab as Docker, and the containerd part can be assumed/implied since it started as a component of docker. I think this would also be better for SEO and someone just scanning through the page, since this tab is still primarily about docker. (also, I just remembered changing the tab name could change a bunch of people's bookmarks if they've bookmarked this, and they'd end up on the k8s tab by default if it gets changed)

Reply:
Sounds good, thanks for looking into it ! I updated the PR accordingly.